src/DTS/UseCase/Render.php

```diff
@@ -33,17 +36,11 @@ public function renderDoc(RenderRequest $request) {
     $this->originalRequest = $request;
     $this->completeRequest = $this->buildRequestOfIds($request);
 
-    $templateId = $this->completeRequest->templateId;
-    $docDataId = $this->completeRequest->dataId;
-
-    throw new \Exception('Dan needs to take it from here...');
-
-    $body = 'Hi Fred';
-
-    $file = tmpfile();
-    fwrite($file, $body);
-    rewind($file);
-    return $file;
+    $body = $this->renderTemplate(
```

Comment:
So I had to dig to find out that this actually returns a path to a file ... so don't call it `$body`